Library/Homebrew/utils/inreplace.rb

```diff
@@ -25,15 +35,17 @@ def inreplace(paths, before = nil, after = nil, audit_result = true)
       errors["`paths` (first) parameter"] = ["`paths` was empty"] if paths.blank?
 
       Array(paths).each do |path|
-        s = File.open(path, "rb", &:read).extend(StringInreplaceExtension)
+        s = (File.open(path, "rb", &:read) || "").extend(StringInreplaceExtension)
```

Comment:
This is a bit hard to follow, could it be two variable declarations instead of one?